recipes/repype/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is in the tarball. No need to add it twice. Please remove it from the PR.

Reply:
Done in 3e73d5308e602ea20b2299aad0d212ef7b26f575 @ocefpaf